src/backend/marsha/core/utils/urls_utils.py

```diff
@@ -0,0 +1,17 @@
+"""Utils for custom path converters."""
+
+from ..views import SUPPORTED_RESOURCES
+
+
+class ResourceConverter:
+    """Url converter to limit the resource choices."""
+
+    regex = "({!s})".format("|".join(m for m in SUPPORTED_RESOURCES))
+
+    def to_python(self, value):
+        """Return the received value as is."""
+        return value
+
+    def to_url(self, value):
+        """Return the received value as is."""
+        return value
```

Comment:
This is not doing much is it?

Reply:
indeed but this is what I found in the django documentation... Should I do something else ?

Reply:
With the reacto made on the `LTIView` I don't need this class anymore.